### PR TITLE
PR: Add using additional modules video to FAQ in docs

### LINKED ISSUE #269: Embed video about connecting additional packages in docs
# Issue Report

## Issue Description

Embed video about connecting additional packages somewhere in docs probably in the FAQ: 
https://www.youtube.com/watch?v=i7Njb3xO4Fw&t=23s

### Other information

* Is the problem specific to documentation for a particular version of Spyder?

* Are you planning on submitting a Github pull request with a suitable change?

* Anything else you'd like to add?

### PR DESCRIPTION
**Pull request by juanis2112**
# Pull Request

## Pull Request Checklist

* [ ] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [ ] Based your PR on the latest version of the correct branch (master or 4.x)
* [ ] Checked your writing carefully for correct English spelling, grammar, etc
* [ ] Described your changes and the motivation for them below
* [ ] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Added video on  using additional module to FAQ in docs.

## Issue(s) Resolved

Fixes #269

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -212,14 +220,8 @@ Using Spyder
 
    Finally, you need to connect Spyder to this environment by changing Spyder’s default Python interpreter. To do this, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
 
-   .. image:: /images/faq/faq-change-environment.png
```
Comment (ccordoba12) [inline doc/faq.rst:215]: If this image is no longer needed, please remove it.

File: doc/faq.rst [hunk 2]
```diff
@@ -212,14 +220,8 @@ Using Spyder
...
-      :alt: Change default environment in Preferences option in status bar
-
    This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section. Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment. If it is not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
 
-   .. image:: /images/faq/faq-python-interpreter-modules.png
```
Comment (ccordoba12) [inline doc/faq.rst:220]: Also this one.

**Review by ccordoba12 — COMMENTED** [review]
Thanks @juanis2112 for this! Besides the comments I left for you below, I'd like to know if you could add a blank space below the video because it looks too crammed to me:

imagen

File: doc/faq.rst (hunk 1 above)
Comment (juanis2112) [reply doc/faq.rst:215]: This is being used somewhere else

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @juanis2112!

**Pull request merged**